--- a/lib/plugin.py
+++ b/lib/plugin.py
@@ -332,7 +332,7 @@
         thumb = listData[0]
         if thumb:
             art_dict = {'icon': thumb, 'thumb': thumb, 'poster': thumb}
-        
+
         plot = listData[1]
 
         listItemFunc = makeListItemClean if ADDON.getSetting('cleanupEpisodes') == 'true' else makeListItem
@@ -1770,7 +1770,7 @@
         if thumb_id.startswith( 'tm-' ):
             return TMDB_IMAGES_URL + '/t/p/w500/' + thumb_id.replace( 'tm-', '' ) + '.jpg'
         return IMAGES_URL + '/catimg/' + thumb_id + '.jpg' + get_thumbnail_headers()
-    
+
     return ''
 
 def solve_media_redirect(url, headers):
